lib/python/rose/bush.py

```diff
@@ -453,6 +454,28 @@ def summary(self, user, form=None):
 
     @cherrypy.expose
     def view(self, user, suite, path, path_in_tar=None, mode=None):
+        template = self.template_env.get_template("view.html")
+
+        data = {}
+        data.update(self._get_suite_logs_info(user, suite))
+        return template.render(
+            rose_version=self.rose_version,
+            script=cherrypy.request.script_name,
+            method="view",
+            time=strftime("%Y-%m-%dT%H:%M:%S+0000", gmtime()),
+            logo=self.logo,
+            title=self.title,
+            host=self.host_name,
+            user=user,
+            suite=suite,
+            path=path,
+            path_in_tar=path_in_tar,
+            mode=mode,
+            **data)
+
+    @cherrypy.expose
+    def file(self, user, suite, path, path_in_tar=None, mode=None,
+             search_string=None, search_mode=SEARCH_MODE_TEXT):
```

Comment:
`file` is a Python built-in, we should avoid using it as a method name. (I know we are breaking this rule elsewhere in the source tree, but they were written before we thought it would be an issue.)